crates/rust-analyzer/src/config.rs

```diff
@@ -479,6 +479,8 @@ config_data! {
         /// How to render the size information in a memory layout hover.
         hover_memoryLayout_size: Option<MemoryLayoutHoverRenderKindDef> = Some(MemoryLayoutHoverRenderKindDef::Both),
 
+        /// Whether to show type bounds from containers for functions.
+        hover_show_containerBounds: bool = true,
```

Comment:
I don't think we want a config for this to be honest

Reply:
So should we display type bounds all the time? I added this config because I’m concerned that some users might find them too noisy in more complex type signature.

Reply:
Yes we are already doing the same for the function bounds themselves so I think this is fine for now, we can add it later if need be